developer-docs-site/docs/concepts/coin-and-token/aptos-token-v2.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Aptos Token"
+id: "aptos-token"
+---
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Aptos Token V2
+
+## Overview of NFTs
+
+An [NFT](https://en.wikipedia.org/wiki/Non-fungible_token) is a non-fungible [token](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token-objects/sources/token.move) 
+or data stored on a blockchain that uniquely defines ownership of an asset. NFTs were first defined in 
+[EIP-721](https://eips.ethereum.org/EIPS/eip-721) and later expanded upon in [EIP-1155](https://eips.ethereum.org/EIPS/eip-1155). 
+NFTs are typically defined using the following properties:
+
+- `name`: The name of the asset. It must be unique within a collection.
+- `description`: The description of the asset.
+- `uri`: A URL pointer to off-chain for more information about the asset. The asset could be media such as an image or video or more metadata.
+- `supply`: The total number of units of this NFT. Many NFTs have only a single supply, while those that have more than one are referred to as editions.
+
+Additionally, most NFTs are part of a collection or a set of NFTs with a common attribute, for example, a theme, creator,
+or minimally contract. Each collection has a similar set of attributes:
+
+- `name`: The name of the collection. The name must be unique within the creator's account.
+- `description`: The description of the collection.
+- `uri`: A URL pointer to off-chain for more information about the asset. The asset could be media such as an image or video or more metadata.
+- `supply`: The total number of NFTs in this collection. 
+- `maximum`: The maximum number of NFTs that this collection can have. If `maximum` is set to 0, then supply is untracked. 
+
+## Design principles
```

Comment:
Can we add a section for comparison against the token v1 standard and why token v2 is better?